ui/cap-react/src/antd/forms/widgets/UriWidget.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
```

Comment:
should we implement somehow like this? with using `BaseInputTemplate` to be more correct
https://github.com/rjsf-team/react-jsonschema-form/blob/main/packages/core/src/components/widgets/URLWidget.tsx

https://github.com/rjsf-team/react-jsonschema-form/blob/main/packages/antd/src/templates/BaseInputTemplate/index.tsx